README.md

```diff
@@ -260,23 +164,7 @@ If you like this project...
 
 > Give credit to everyone who inspired your codebase.
 
-I would like to thank...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
-
-## ❓ FAQ <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+I would like to thank the microverse staff for doing their best to equip us with the necesarry skills to start our software engineering carreers. I would also like to thank my coding partners who have helped me along the way.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Good job creating your README file , however to keep it professional you should avoid broken link 👍 . A broken link is attached to your MIT link 😄 

- [ ] Kindly remove the `Note` text below the MIT link to keep it professional 👍 

![Capture](https://github.com/ismayito/vet-clincic-database/assets/80458199/2d2cd16e-5807-45b7-a05d-e1854388ef83)